package-builders/Dockerfile.debian_bookworm

```diff
@@ -0,0 +1,50 @@
+FROM debian:bookworm
```

Comment:
This is not an [available image tag](https://hub.docker.com/_/debian?tab=description&page=1&name=bookwork) from the Debian upstream repo:

```bash
prologic@Jamess-iMac
Tue Jan 14 08:28:09
~/NetData/helper-images
 (debian-builder-images) 0
$ docker build -t netdata/packager:debian_bookworm -f package-builders/Dockerfile.debian_bookworm .
Sending build context to Docker daemon  288.8kB
Step 1/13 : FROM debian:bookworm
manifest for debian:bookworm not found: manifest unknown: manifest unknown
```

* Did you do something special to get this on your system?
* Are we supporting Debian bookworm?
  * If so we're going to have to also handle this somehow...

Reply:
Bookworm is the tentative codename for Debian 12. it's not yet available because 11 is not out yet. Only reason I had it in here is that we've got it listed in the os-test images as well. I'm assuming that the intent is to just need to enable a build rule when it gets released instead of having to add new files, but I can remove it if desired.

Reply:
My inclination is to remove it for now until we need it;

or:

Build our own `debian_bookworm` image ourselves.

I just happened to try/test that one by accident really and it doesn't work so it kind of threw me off 🤣